packages/auth-construct/README.md

```diff
@@ -0,0 +1,135 @@
+# AmplifyAuth Construct
+
+This package vends an L3 CDK Construct wrapping the behavior of Amplify Auth. This enables quick development and interation with AWS Cognito.
+
+The primary way to use this construct is to invoke it with a configuration object. You can declare the individual settings for your authentication by passing them as properties to the AmplifyAuth construct.
+
+Note: only a single instance of the `AmplifyAuth` construct can be invoked within a CDK synthesis at this point in time.
```

Comment:
Do we validate this at runtime ?

Reply:
yes it will fail at cdk synth 
![Uploading image.png…]()